### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by yourgirlcodes**
WIP for CLI CR - 
working on direct appplication and segments, update in audiences for cleaner code

### CODE REVIEW

**Comment by yourgirlcodes** [comment]
PR for updates before demo

**Comment by iBaryo** [comment]
so this can be closed for the next PR?
https://github.com/iBaryo/gigya-cdp-cli/pull/7

**Comment by yourgirlcodes** [comment]
Newest for CR

File: defaultSchemaFakers.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"config":{"primaryPhone":"tyre","primaryEmail":"name"},"password":""}
```
Comment (iBaryo) [inline defaultSchemaFakers.json:1]: remove this file

File: boilerplateFakers.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"config":{"primaryEmail":"email","primaryPhone":"phoneNumber","purchasePrice":"price","orderId":"uuid","sessionTime":"number","pageUrl":"url"},"password":""}
```
Comment (iBaryo) [inline boilerplateFakers.json:1]: remove this file

**Review by iBaryo — CHANGES_REQUESTED** [review]

**Pull request merged**